website/docs/development/documentation.md

```diff
@@ -25,6 +25,7 @@ issue number with one of the following extensions:
 * `docs` : Addition or updates to documentation
 * `config` : Changes or addition to the configuration structure
 * `maintenance` : Changes that improve the maintainability of the code
+* `trivial`: Minor Changes
```

Comment:
I am actually not sure what purpose trivial news fragments are serving.

Reply:
`trivial` is already supported, just not documented. Should we remove the support for `trivial` ?

Reply:
Yeah, let's just remove it.

Reply:
Done in https://github.com/facebookresearch/hydra/pull/1385/commits/eb893120149d00a66699c7c57f55d616dcb99818